libmamba/src/core/package_fetcher.cpp

```diff
@@ -324,26 +325,12 @@ namespace mamba
 
     std::string PackageFetcher::channel() const
     {
-        if (m_package_info.package_type == specs::PackageType::Conda)
```

Comment:
Not sure about why this was added but the package type is set to its default value `Unknown` while it should be set to `Conda` when downloading using OCI mirrors.
That said, changing this (in this PR) still makes the tests pass in the CI, so I don't know what's the use case of the `else`...
Same for `url_path()`.

Reply:
I don't remember either Oo